tokio-util/src/task/join_map.rs

```diff
@@ -96,11 +92,10 @@ use tokio::task::{AbortHandle, Id, JoinError, JoinSet, LocalSet};
 /// ```
 ///
 /// [`JoinSet`]: tokio::task::JoinSet
-/// [unstable]: tokio#unstable-features
 /// [abort]: fn@Self::abort
 /// [abort_matching]: fn@Self::abort_matching
 /// [contains]: fn@Self::contains_key
-#[cfg_attr(docsrs, doc(cfg(all(feature = "rt", tokio_unstable))))]
+#[cfg_attr(docsrs, doc(cfg(feature = "rt")))]
 pub struct JoinMap<K, V, S = RandomState> {
```

Comment:
Once you add the `join_map` feature, this annotation may not be necessary. Please try removing it and checking the docs.